### ISSUE TITLE
US recovered cases are all 0 in the time file

### ISSUE THREAD

**Issue by MMogollon11** [CSSEGISandData/COVID-19#1054]
US recovered cases are all 0 in the time file for March 18
![Chart 1 US States Summary 03182020C](https://user-images.githubusercontent.com/60978422/77094221-2bfcdb80-69da-11ea-87c4-cd9edfd7857f.jpg)

**Comment by jclerman** [comment]
This is true for both the daily reports file for March 18, and for the March-18 number in the time-series file.  Numbers for earlier dates in the time-series file are nonzero.

**Comment by yetzt** [comment]
Might be connected to #1022
